.github/workflows/ci.yml

```
@@ -26,17 +26,25 @@ jobs:
         with:
           persist-credentials: false
-      # rustfmt.toml enables `wrap_comments`, an unstable option that only the
-      # nightly rustfmt applies — stable silently ignores it. Run nightly here so
-      # the check actually enforces comment wrapping. Local devs: `cargo +nightly fmt`.
-      - uses: dtolnay/rust-toolchain@nightly
+      - uses: dtolnay/rust-toolchain@master
         with:
+          toolchain: ${{ env.RUSTFMT_NIGHTLY }}
           components: rustfmt
-      - run: cargo +nightly fmt --all -- --check
+      - run: cargo +$RUSTFMT_NIGHTLY fmt --all -- --check
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:1:8: Double quote to prevent globbing and word splitting

(shellcheck)